Add "No Oracle DB hostname reported" to known issues (#28098)
DataDog/documentation: content/en/database_monitoring/setup_oracle/troubleshooting.md

### "Table or view does not exist" error in `agent.log`
Execute the permission grants listed in the **Grant permissions** step of the [setup instructions][3] for your hosting type.

## Known issues

### No Oracle DB hostname reported

> The Datadog Agent detects Oracle DB hostname by running SQL command against [V$INSTANCE][4].

Reviewer: ```suggestion
The Datadog Agent detects Oracle DB hostnames by running SQL commands against [V$INSTANCE][4].
```

Reviewer: Thanks for flagging that, followed up on the comment.

> When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. This behavior has been confirmed with Oracle Autonomous Database.

Reviewer: ```suggestion
When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB hostname as empty. This behavior has been confirmed with Oracle Autonomous Database.
```

### "Table or view does not exist" error in `agent.log`
Execute the permission grants listed in the **Grant permissions** step of the [setup instructions][3] for your hosting type.

## Known issues

### No Oracle DB hostname reported

> Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].

Reviewer: ```suggestion
The Datadog Agent detects Oracle DB hostnames by running SQL commands against [V$INSTANCE][4].
```

Reviewer 2: Should `hostnames` and `commands` be singular as the Agent runs only one query which returns only one hostname?

Reviewer: Thanks for that question, and apologies for the delay following up here. In this case, no, they should not be singular, as what's being referenced is a general behavior rather than a single occurrence of a behavior. 

If you feel that it's important to emphasize that the Agent runs only one query, which returns only one hostname, then an alternative phrasing would be:
The Datadog Agent detects **the** Oracle DB hostname by running **a** SQL command against [V$INSTANCE]

> When Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. We have confirmed this behavior with Oracle Autonomous Database.

Reviewer: ```suggestion
When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. This behavior has been confirmed with Oracle Autonomous Database as well.
```

Reviewer 2: Shall we leave `as well` out? We have confirmed the behaviour *only* with Oracle Autonomous Database.

> In this case, we recommend setting the `reported_hostname` in the `conf.yaml`.

Reviewer: ```suggestion
In this case, Datadog recommends setting the `reported_hostname` in the `conf.yaml` file.
```

> [Since v7.65.0 and above][5], the Datadog Agent will output logs if this condition is met.

Reviewer: ```suggestion
[Since v7.65.0 and later][5], the Datadog Agent outputs logs if this condition is met.
```

[1]: /database_monitoring/setup_oracle/
[2]: /database_monitoring/setup_oracle#recommended-agent-version
[3]: /database_monitoring/setup_oracle#setup
[4]: https://docs.oracle.com/en/database/oracle/oracle-database/23/refrn/V-INSTANCE.html

> [5]: https://github.com/DataDog/datadog-agent/pull/34866

Reviewer: ```suggestion
[5]: https://github.com/DataDog/datadog-agent/releases/tag/7.65.0
```
